libs/sdk-backend-tiger/src/convertors/fromBackend/WorkspaceConverter.ts

```diff
@@ -9,7 +9,7 @@ export const workspaceConverter = ({
 }: JsonApiWorkspaceOut | JsonApiWorkspaceOutWithLinks): IWorkspaceDescriptor => {
     const parentWorkspace = relationships?.parent?.data?.id;
     return {
-        description: attributes?.name || id,
+        description: attributes?.description || id,
```

Comment:
I think we should keep using `attributes?.name`, to prevent possible regression bugs:

```
description: attributes?.description || id,
```

↓

```
description: attributes?.description || attributes?.name || id,
```

Reply:
I updated it thanks